packages/vscode/src/plugins/prisma-postgres-manager/PrismaPostgresTreeDataProvider.ts

```diff
@@ -176,7 +179,12 @@ class PrismaRemoteDatabaseItem extends vscode.TreeItem {
     command: 'prisma.studio.launchForDatabase',
     title: 'Launch Prisma Studio',
     arguments: [
-      { type: 'remote', workspaceId: this.workspaceId, projectId: this.projectId, databaseId: this.databaseId },
+      {
+        type: 'remote',
+        workspaceId: this.workspaceId,
+        projectId: this.projectId,
+        databaseId: this.databaseId,
+      } as LaunchArgRemote,
```

Comment:
Why `as` and not `satisfies` as for the local DB one? 🤔 
```suggestion
      } satisfies LaunchArgRemote,
```